cpg_infra/billing_aggregator/aggregate/hail.py

```diff
@@ -62,6 +62,26 @@ def get_billing_projects():
     return ds
 
 
+def infer_batch_namespace(batch: dict) -> str:
+    """
+    Infer the namespace from the batch attributes
+    """
+    namespace = batch.get('attributes', {}).get('namespace')
+    user = batch.get('user')
+    default = None
+    if namespace:
+        return namespace
+    elif user:
```

Comment:
elif -> if 
would be enough here, we already exit the function on previous "if" above.

Reply:
Changed!